packages/coinstac-ui/app/render/components/data-discovery/edit-dataset/index.js

```diff
@@ -0,0 +1,3 @@
+import EditDataset from './edit-dataset';
+
+export default EditDataset;
```

Comment:
whats the idea behind this?

Reply:
The idea here is to have the `edit-dataset.jsx` file, so it's easier to search. Also I'd have an `index.js` file to make it easier to import the component in other files.